controllers/array_action/array_mediator_interface.py

```diff
@@ -368,7 +368,7 @@ def delete_host(self, host_name):
            host_name : name of the host in the storage system
 
         Returns:
-            None
+           None
 
         Raises:
             None
```

Comment:
these `None`s don't seem to be aligned with each other

Reply:
done - #561